packages/terra-dropdown-button/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "terra-dropdown-button",
+  "main": "lib/DropdownButton.js",
+  "version": "0.0.0",
+  "description": "A button that, when pressed, opens a dropdown with more buttons",
```

Comment:
Lets update this to describe this gives access to a dropdown and split button components for selecting a single option (action) form a list of options.